### PR TITLE
[Doc] Use substrate-ssvm docker image.

### PR DESCRIPTION
**Pull request by dm4**

### CODE REVIEW

**Review by hydai — CHANGES_REQUESTED** [review]
Please add our demo video.

**Review by hydai — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,207 +2,32 @@
...
 
-Our achievement is divided into the following serval parts.
+Our achievement is divided into the following several parts.
 
-1. Provdie a [EWASM Test Guide](https://github.com/second-state/rust-ssvm/docs/EWASM-TEST.md) show how to use official EWASM test suite verify our EWASM engine (SSVM).
+1. Provdie a [EWASM Test Guide](https://github.com/second-state/rust-ssvm/blob/master/docs/EWASM_TEST.md) show how to use official EWASM test suite verify our EWASM engine (SSVM).
 2. Create [rust-ssvm](https://github.com/second-state/rust-ssvm), it's SSVM with a rust interface through EVMC.
 3. Create a SRML [pallet-ssvm](https://github.com/second-state/pallet-ssvm) use rust-ssvm as substrate node's EWASM engine.
 4. Integrate pallet-ssvm to [substrate-ssvm-node](https://github.com/second-state/substrate-ssvm-node).
 5. Extend [command line tool](https://github.com/second-state/substrate-cli-tools) for support substrate-ssvm-node just like evm pallet did.
```
Comment (CaptainVincent) [inline README.md]: Help me plus `s` after tool.

File: README.md [hunk 2]
```diff
@@ -2,207 +2,32 @@
...
-(2 matching events received)
-```
-
-Bob have 3 ERC20 tokens.
-
+See our documents for more information about interacting with node:
+- [Use command line tools](./docs/interact-using-cli.md)
+  - [Demo video](https://drive.google.com/open?id=149AgZkvXeQZEAlZlNQ7bcZ8V2zCjpQf6)
+- [Use Substrate web interface](./docs/interact-using-web.md)
+  - [Demo video](https://drive.google.com/open?id=119L2oCVuaGAZJ1yQj3YVsKAepc0DcFq-)
```
Comment (CaptainVincent) [inline README.md]: personal suggestion format.
- Use command line tools
   - [Demo video](https://drive.google.com/open?id=149AgZkvXeQZEAlZlNQ7bcZ8V2zCjpQf6)
   - [Doc](./docs/interact-using-cli.md)
 - Use Substrate web interface
   - [Demo video](https://drive.google.com/open?id=119L2oCVuaGAZJ1yQj3YVsKAepc0DcFq-)
   - [Doc](./docs/interact-using-web.md)

File: docs/interact-using-cli.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+(docker) cd ~/node/substrate-cli-tools/typescript
+(docker) yarn install && yarn run tsc
+(docker) chmod +x dist/*.js
+```
+
+## Run
+
+After building `substrate-ssvm-node`, we could start a Substrate node that supports Ewasm by our [pallet-ssvm](https://github.com/second-state/pallet-ssvm):
+
+```bash
```
Comment (CaptainVincent) [inline docs/interact-using-cli.md:41]: Maybe here need annotations execute inside ~/node/substrate-ssvm-node folder. 
(Discontinuous with before command)

File: docs/interact-using-web.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+```bash
+> git clone https://github.com/second-state/substrate-ssvm-node.git
+> docker run -it --rm \
+  --name ssvm \
+  -p 9944:9944 \
+  -v $(pwd):/root/node \
+  -w /root/node/substrate-ssvm-node \
+  secondstate/substrate-ssvm:latest \
+  cargo run --release --bin node-template -- --dev
```
Comment (CaptainVincent) [inline docs/interact-using-web.md]: Does here need add `--ws-external`?

File: README.md (hunk 1 above)
Comment (dm4) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (dm4) [reply README.md]: good suggestion!

File: docs/interact-using-cli.md (hunk 3 above)
Comment (dm4) [reply docs/interact-using-cli.md:41]: sure

File: docs/interact-using-web.md (hunk 4 above)
Comment (dm4) [reply docs/interact-using-web.md]: yes, I missed that QQ

**Comment by CaptainVincent** [comment]
LGTM.

**Pull request merged**